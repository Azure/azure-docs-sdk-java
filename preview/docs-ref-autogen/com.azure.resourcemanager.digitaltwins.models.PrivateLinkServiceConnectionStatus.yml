### YamlMime:JavaType
uid: "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus"
fullName: "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus"
name: "PrivateLinkServiceConnectionStatus"
nameWithType: "PrivateLinkServiceConnectionStatus"
summary: "Defines values for PrivateLinkServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.digitaltwins.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-digitaltwins:1.0.0-beta.1
