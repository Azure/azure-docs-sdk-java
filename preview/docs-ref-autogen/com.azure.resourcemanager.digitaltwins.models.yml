### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.digitaltwins.models"
fullName: "com.azure.resourcemanager.digitaltwins.models"
name: "com.azure.resourcemanager.digitaltwins.models"
summary: "Package containing the data models for AzureDigitalTwinsManagementClient. Azure Digital Twins Client for managing\n DigitalTwinsInstance."
classes:
- "com.azure.resourcemanager.digitaltwins.models.AuthenticationType"
- "com.azure.resourcemanager.digitaltwins.models.CheckNameRequest"
- "com.azure.resourcemanager.digitaltwins.models.ConnectionProperties"
- "com.azure.resourcemanager.digitaltwins.models.ConnectionPropertiesPrivateEndpoint"
- "com.azure.resourcemanager.digitaltwins.models.ConnectionPropertiesPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.digitaltwins.models.ConnectionPropertiesProvisioningState"
- "com.azure.resourcemanager.digitaltwins.models.ConnectionState"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescriptionListResult"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResourceListResult"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResourceProperties"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsIdentity"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsIdentityType"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsPatchDescription"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsPatchProperties"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsResource"
- "com.azure.resourcemanager.digitaltwins.models.EndpointProvisioningState"
- "com.azure.resourcemanager.digitaltwins.models.EndpointType"
- "com.azure.resourcemanager.digitaltwins.models.EventGrid"
- "com.azure.resourcemanager.digitaltwins.models.EventHub"
- "com.azure.resourcemanager.digitaltwins.models.ExternalResource"
- "com.azure.resourcemanager.digitaltwins.models.GroupIdInformationProperties"
- "com.azure.resourcemanager.digitaltwins.models.OperationDisplay"
- "com.azure.resourcemanager.digitaltwins.models.OperationListResult"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpoint"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnectionProperties"
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus"
- "com.azure.resourcemanager.digitaltwins.models.ProvisioningState"
- "com.azure.resourcemanager.digitaltwins.models.PublicNetworkAccess"
- "com.azure.resourcemanager.digitaltwins.models.Reason"
- "com.azure.resourcemanager.digitaltwins.models.ServiceBus"
interfaces:
- "com.azure.resourcemanager.digitaltwins.models.CheckNameResult"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwins"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.Definition"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.Blank"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithPrivateEndpointConnections"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.DefinitionStages.WithTags"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.Update"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.UpdateStages"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.UpdateStages.WithProperties"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsDescription.UpdateStages.WithTags"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.Definition"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.DefinitionStages"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.DefinitionStages.Blank"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.Update"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.UpdateStages"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpointResource.UpdateStages.WithProperties"
- "com.azure.resourcemanager.digitaltwins.models.DigitalTwinsEndpoints"
- "com.azure.resourcemanager.digitaltwins.models.GroupIdInformation"
- "com.azure.resourcemanager.digitaltwins.models.GroupIdInformationResponse"
- "com.azure.resourcemanager.digitaltwins.models.Operation"
- "com.azure.resourcemanager.digitaltwins.models.Operations"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnection.UpdateStages.WithProperties"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.digitaltwins.models.PrivateEndpointConnectionsResponse"
- "com.azure.resourcemanager.digitaltwins.models.PrivateLinkResources"
metadata: {}
package: "com.azure.resourcemanager.digitaltwins.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-digitaltwins:1.0.0-beta.1
