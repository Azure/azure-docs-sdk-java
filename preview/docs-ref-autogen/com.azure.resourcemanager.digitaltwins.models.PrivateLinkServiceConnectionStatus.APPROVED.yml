### YamlMime:JavaMember
uid: "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.digitaltwins.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.digitaltwins.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-digitaltwins:1.0.0-beta.1
